frontend/themes/gov-uk/client-js/govuk-cookies.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable no-undef, no-param-reassign, no-unused-vars */
```

Comment:
this is clientside govuk stuff so I can't mess around with adding eslint requests for things